File: Java/src/com/codesignal/c01_intro/E01Add.java
package com.codesignal.c01_intro;

import java.util.Scanner;

public class E01Add {

    public static int addNumbers(int param1, int param2) {
        return param1 + param2;
    }

    public static void main(String[] args) {

        int[][] testCases = {
            {1, 2},
            {0, 1000},
            {2, -39},
            {99, 100},
            {-100, 100},
            {-1000, -1000}
        };

        for (int[] testCase : testCases) {
            int result = addNumbers(testCase[0], testCase[1]);
            System.out.printf("Input: %d %d Output: %d%n", testCase[0], testCase[1], result);
        }

    }
}

File: C++/c01_intro/e01_add.cpp
#include <iostream>

using namespace std;

int add(int param1, int param2) {
    return param1 + param2;
}

int main() {
    int param1, param2;
    cin >> param1 >> param2;
    cout << add(param1, param2) << endl;
    return 0;
}
